Dockerfile

```diff
@@ -1,14 +1,11 @@
-#
-# Build and copy GraphQL binaries and run them.
-#
-# Version values referenced from https://hub.docker.com/_/microsoft-dotnet-aspnet
-FROM mcr.microsoft.com/dotnet/sdk:6.0 as build
+FROM mcr.microsoft.com/dotnet/sdk:6.0-cbl-mariner2.0. AS build
+
 
 WORKDIR /src
-COPY [".", "./"]
+COPY . .
```

Comment:
nit: could you please ensure the contents are in the right folder similar to the previous COPY syntax? This could be part of testing that can be added to the `How was this tested?` section